### PR TITLE
final comments decision

### PR DESCRIPTION
**Pull request by awalterschulze**

### CODE REVIEW

File: doc/FormattingDecisionComments.md [hunk 1]
```diff
@@ -116,11 +131,17 @@ We have also [seen](https://github.com/erlang/otp/blob/master/lib/stdlib/src/erl
...
-- ✅ Minimize diff when changing a single line
+ - ✅ Minimize diff when changing a single line
+ - ❌ Currently popular style
+ - ❌ Welcoming to new comers
 
 The standard erlang mode for emacs puts the single quoted comments by default at position 48 on the line.
 This means that comments are not realigned based on line lengths, but always aligned at the same fixed column.
 
+When speaking to people this seemed to be a relatively popular style,
+but after conducting some analyses, it seems there wasn't a consensus on a fixed position for comments to be aligned too.
```
Comment (elbrujohalcon) [inline doc/FormattingDecisionComments.md:142]:
```suggestion
but after conducting some analyses, it seems there wasn't a consensus on a fixed position for comments to be aligned to.
```

File: doc/FormattingDecisionComments.md [hunk 2]
```diff
@@ -2,7 +2,15 @@
...
-## Conventions in Documentation
+Currently `erlfmt` moves all trailing comments to the line above, so that all comments are always alone on their own line.
+
+After looking at documentation and doing an analyses we have concluded that [directly following comments](#single-comments-on-the-same-line) should also be supported in future.
+This does not include aligning comments, since:
+  - we could not find a popular enough convention of aligning to a specific column number and
+  - aligning comments can cause large diffs, from changing a single line for comments to be re-aligned.
+More details on this future change can be found in issue [#219](https://github.com/WhatsApp/erlfmt/issues/219)
+
+## Documentation
```
Comment (elbrujohalcon) [inline doc/FormattingDecisionComments.md:13]: You can also add [this guideline](https://github.com/inaka/erlang_guidelines#comment-levels) to the documentation examples in your list.

Comment (awalterschulze) [reply doc/FormattingDecisionComments.md:13]: Thank you, great idea :)

File: doc/FormattingDecisionComments.md (hunk 1 above)
Comment (awalterschulze) [reply doc/FormattingDecisionComments.md]: Fixed :)
Good catch

**Review by michalmuskala — APPROVED** [review]

**Pull request merged**